### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
Relates #23 
This PR contains the `'LandingPage'` layout.
to render this layout use `'/'`

### CODE REVIEW

File: client/src/Components/App.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 // import components
-import Home from "./Layouts/Home";
+import LangingPge from "./Layouts/LangingPge";
```
Comment (arrested-developer) [inline client/src/Components/App.js]: spelling error - please correct to LandingPage

File: client/src/Components/App.js [hunk 2]
```diff
@@ -20,42 +20,41 @@ class App extends Component {
...
-    </BrowserRouter>
-  );
-}
+  render() {
+    const { currentSession } = this.state;
+    return (
+      <BrowserRouter>
+        <div className="App">
+          <Switch>
+            <Route path="/" exact component={LangingPge} />
```
Comment (arrested-developer) [inline client/src/Components/App.js]: LandingPage

File: client/src/Components/Layouts/LangingPge/LangingPge.test.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+test("LandingPage matches snapshot", () => {
+  const context = {};
+
+  const component = renderer.create(
+    <StaticRouter context={context}>
+      <LandingPage />
+    </StaticRouter>,
+  );
+  expect(component).toMatchSnapshot();
```
Comment (arrested-developer) [inline client/src/Components/Layouts/LangingPge/LangingPge.test.js]: This fails when I run locally

screen shot 2018-11-26 at 12 40 06

File: client/src/Components/Layouts/LangingPge/LangingPge.test.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
```
Comment (arrested-developer) [inline client/src/Components/Layouts/LangingPge/LangingPge.test.js]: Please rename file and folder to correct spelling:

LandingPage

File: client/src/Components/Layouts/LangingPge/LangingPge.test.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/Components/Layouts/LangingPge/LangingPge.test.js]: @arrested-developer 
I ran that locally and the test passed, could you check that again, please

File: client/src/Components/Layouts/LangingPge/LangingPge.test.js (hunk 4 above)
Comment (RamyAlshurafa) [reply client/src/Components/Layouts/LangingPge/LangingPge.test.js]: I shouldn't code in the dark anymore :smile:

**Pull request merged**